### PR TITLE
Fixes FD calculation within AFNI HMC workflow

### PR DESCRIPTION
**Pull request by jvarada**
Removes dependency from custom fd calculation within AFNI HMC workflow. Assigns Nipype's FD node the correct output from 3dVolreg (oned_file) which contains the motion parameters.

### CODE REVIEW

**Comment by oesteban** [comment]
The tests are failing because google drive is taking down the access to those data. In master you have the new version that fetches from OSF, it should work fine. Therefore, please merge in upstream master.

**Comment by jvarada** [comment]
Done

File: mriqc/workflows/utils.py [hunk 1]
```diff
@@ -35,72 +35,3 @@ def fwhm_dict(fwhm):
     fwhm = [float(f) for f in fwhm]
     return {'x': fwhm[0], 'y': fwhm[1],
             'z': fwhm[2], 'avg': fwhm[3]}
-
-def fd_jenkinson(in_file, rmax=80., out_file=None):
```
Comment (oesteban) [inline mriqc/workflows/utils.py]: why removing this? I don't think ```3dvolreg``` outputs parameters according to the FD definition. This conversion might be broken, but I don't think removing this is a good idea.

Comment (oesteban) [inline mriqc/workflows/utils.py]: Ok sorry. I'm wrong. The question here is if the parameters that ```3dvolreg``` writes are equivalent to the ```plots``` that ```mcflirt``` computes.

Comment (oesteban) [inline mriqc/workflows/utils.py]: Ok, the question here is where is the center of rotation. AFAIK, ```mcflirt``` sets the center of rotation in the _center of masses_ of the brain mask.

Comment (oesteban) [inline mriqc/workflows/utils.py]: sounds good :)

**Comment by jvarada** [comment]
@oesteban - The changes required quite a bit of rewiring, please check the commit log for a description of the changes I made, and let me know if you have any questions.

**Comment by jvarada** [comment]
@oesteban Have you had a chance to look at this?

**Comment by jvarada** [comment]
Merged the newest mriqc master into this branch, and resolved merge conflicts.

**Pull request merged**